specification/synapse/data-plane/Microsoft.Synapse/preview/2019-11-01-preview/roleAssignments.json

```diff
@@ -0,0 +1,356 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2019-11-01-preview",
+    "title": "SynapseClient"
+  },
+  "x-ms-parameterized-host": {
+    "hostTemplate": "{workspaceName}.{SynapseDnsSuffix}",
+    "parameters": [
+      {
+        "name": "workspaceName",
+        "description": "The name of the workspace to execute operations on.",
+        "required": true,
+        "type": "string",
+        "in": "path",
+        "x-ms-skip-url-encoding": true
+      },
+      {
+        "$ref": "#/parameters/SynapseDnsSuffixInPath"
+      }
+    ]
+  },
+  "schemes": [
+    "https"
+  ],
+  "paths": {
+    "/rbac/roleAssignments": {
+      "post": {
+        "tags": [
+          "RoleAssignments"
+        ],
+        "operationId": "RoleAssignments_CreateRoleAssignment",
+        "description": "Create role assignment.",
+        "x-ms-examples": {
+          "Create role assignment": {
+            "$ref": "./examples/RoleAssignments_CreateRoleAssignment.json"
+          }
+        },
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "request",
+            "in": "body",
+            "required": true,
+            "description": "Details of role id and object id.",
+            "schema": {
+              "$ref": "#/definitions/RoleAssignmentRequest"
+            }
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success response.",
+            "schema": {
+              "$ref": "#/definitions/RoleAssignmentDetails"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorContract"
+            }
+          }
+        }
+      },
+      "get": {
+        "tags": [
+          "RoleAssignments"
+        ],
+        "operationId": "RoleAssignments_ListRoleAssignments",
+        "description": "List role assignments.",
+        "x-ms-examples": {
+          "Get role assignment information": {
+            "$ref": "./examples/RoleAssignments_ListRoleAssignments.json"
+          }
+        },
+        "consumes": [],
+        "produces": [
+          "application/json",
+          "text/json"
```

Comment:
Sorry I meant to remove them all, I thought I was clear sorry :(

Reply:
> Sorry I meant to remove them all, I thought I was clear sorry :(

Sorry, I have removed all text/json